docker/README.md

```diff
@@ -134,12 +134,12 @@ Please review these changes in the
 [legacy-auth/env.template](legacy-auth/env.template)
 and in the appropriate `legacy-auth/compose*` files.
 
-To upgrade from versions prior to FiftyOne Teams v1.6
+To upgrade from versions prior to FiftyOne Teams v1.6 (see also [Upgrade Process Recommendations](#upgrade-process-recommendations)):
```

Comment:
This section is pretty duplicative of the "Upgrade Process Recommendations" section below. Maybe it's possible to remove or simplify. Currently I have the linked in both directions, I sort of get why we list the updated envvars etc here in the "FiftyOne Teams Features" section